Sources/Plasma/PubUtilLib/plAvatar/plArmatureMod.h

```diff
@@ -234,9 +235,9 @@ class plArmatureMod : public plArmatureModBase
 
     virtual bool ToggleDontPanicLinkFlag() { fDontPanicLink = fDontPanicLink ? false : true; return fDontPanicLink; }
 
-    int GetBrainCount();
+    size_t GetBrainCount();
```

Comment:
```suggestion
    size_t GetBrainCount() const;
```

Could also be inlined.